test/aria/utils/Html.js

```diff
@@ -51,6 +51,14 @@ Aria.classDefinition({
                                 this.assertEquals(got, value, "data-" + dataKey + " should be %2, got %1");
                             }
                         }
+                    } else if (key === "aria") {
+                        for (var ariaKey in attribute) {
+                            if (attribute.hasOwnProperty(ariaKey)) {
+                                var value = stringUtil.encodeForQuotedHTMLAttribute(attribute[ariaKey]);
```

Comment:
It's that in general this test re-implements the code itself, I am not a big fan...